manager/rbac_manager.py

```diff
@@ -237,6 +237,7 @@ class RbacRoutePermissions:
     ADVANCED_REPORTING = [[RbacPermission(RbacApp.VULNERABILITY, RbacResource.ADVANCED_REPORT, RbacAction.READ), INVENTORY_READ]]
     REPORT_AND_EXPORT = [[RbacPermission(RbacApp.VULNERABILITY, RbacResource.EXPORT_AND_REPORT, RbacAction.READ), INVENTORY_READ]]
     REMEDIATIONS_READ = [[RbacPermission(RbacApp.REMEDIATIONS, RbacResource.REMEDIATION, RbacAction.READ), INVENTORY_READ]]
+    VULNERABILITY_WRITE = [[RbacPermission(RbacApp.VULNERABILITY, RbacResource.ANY, RbacAction.WRITE)]]
```

Comment:
Is the list needed?

Put it next to VULNERABILITY_ANY, to have it grouped

Reply:
I don't understand what you want but `INVENTORY_READ` is missing there, I'll add it

Reply:
ok